ProphetBot/helpers/entity_helpers.py

```diff
@@ -3,6 +3,7 @@
 import random
 from datetime import datetime
 from statistics import mean
+from typing import List
```

Comment:
Looks like this import isn't used. Should probably delete it if that's the case.